lib/quantizeAttributes.js

```diff
@@ -0,0 +1,348 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+module.exports = quantizeAttributes;
+
+/**
+ *
+ * @param gltf
+ * @param options
+ *
+ * @returns glTF with quantized attributes
+ *
+ * @example
+ * // Quantize the positions of a simple mesh
+ * var gltf = {
+ *     "accessors": {
+ *         "accessor_0": {...}
+ *         "accessor_1": {...}
+ *     },
+ *     "meshes": {
+ *         "geometry": {
+ *             "name": "Mesh",
+ *             "primitives": [
+ *                 {
+ *                     "attributes": {
+ *                         "NORMAL": "accessor_0",
+ *                         "POSITION": "accessor_1"
+ *                     }
+ *                 }
+ *             ]
+ *         }
+ *     }
+ * };
+ * var options = {
```

Comment:
`options` allows you to specify which primitives you want quantized. I'll make the documentation more robust. It's really just for generating test models though, in practice I think you would probably want to quantize everything possible, which is the default behavior if options is undefined.